var/spack/repos/builtin/packages/py-cartopy/package.py

```diff
@@ -12,26 +12,35 @@ class PyCartopy(PythonPackage):
     homepage = "http://scitools.org.uk/cartopy/"
     url      = "https://github.com/SciTools/cartopy/archive/v0.16.0.tar.gz"
 
+    version('0.17.0', sha256='137642e63952404ec0841fa0333ad14c58fbbf19cca2a5ac6a38498c4b4998fb')
     version('0.16.0', sha256='cadf62434492c965220b37f0548bc58180466ad6894a1db57dbc51cd43467e5c')
 
     depends_on('py-setuptools@0.7.2:', type='build')
     depends_on('py-cython@0.15.1:',    type='build')
+    depends_on('py-futures',        type=('build', 'run'), when='^python@:2')
```

Comment:
I don't see this listed anywhere

Reply:
Indeed, not required for now. Will probably be required for the next version: https://github.com/SciTools/cartopy/blob/master/requirements/default.txt#L6